gpgs/api/gpg.script_api

```diff
@@ -95,6 +96,46 @@
             end
             ```
 
+#*****************************************************************************************************
+
+  - name: get_id_token
+    type: function
+    desc: Get the current GPGS player id token. Available only if "gpgs.client_id" is configured in game.project
+          and "gpgs.request_id_token = 1".
+
+    returns:
+      - name: id_token
+        type: string
+        desc: The player ID token.
+
+    examples:
+      - desc: |-
+            ```lua
+            if gpgs then
+              local id_token = gpgs.get_id_token()
+            end
+            ```
+
+#*****************************************************************************************************
+
+  - name: get_server_auth_code
+    type: function
+    desc: Get the current GPGS player id token. Available only if "gpgs.client_id" is configured in game.project
+          and "gpgs.request_server_auth_code = 1".
+
+    returns:
+      - name: server_auth_code
+        type: string
+        desc: The server auth code for logged in account.
+
+    examples:
+      - desc: |-
+            ```lua
+            if gpgs then
+              local server_auth_code = gpgs.get_server_auth_code()
+            end
+            ```
+
 #*****************************************************************************************************
 
```

Comment:
Could you also update https://github.com/defold/extension-gpgs/blob/master/docs/_data/api.yml , please?

Reply:
Done